assignments/callbacks.js

```diff
@@ -23,30 +23,115 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
```

Comment:
Add console logs because how else are you checking that these work?

Reply:
I use the console almost exclusively then copy and paste into vs code. Been working with quokka more recently but with the call backs just had a few arrays set up and would call the function and pass it in and in the console you don’t need to explicitly type out console.log to check a function